src/app/shared/components/add-cluster-from-template-dialog/component.ts

```diff
@@ -53,6 +56,8 @@ export class AddClusterFromTemplateDialogComponent implements OnInit, OnDestroy
   ) {}
 
   ngOnInit(): void {
+    this.quotaWidget = this.data.quotaWidget;
```

Comment:
This component (dialog) can be opened from more than one place in the UI so are we passing `quotaWidget` property from everywhere?

Reply:
No, the widget will only be visible when it is passed to the dialog's data.